src/dao/pages/ProposalDetailsPage/ProposalDetailsComponents/ProposalActionDetails.tsx

```diff
@@ -24,7 +26,9 @@ export function ProposalActionDetails(props: ProposalActionDetailsProps) {
           <HederaIcon />
           <SendTokenIcon boxSize={5} stroke={Color.Destructive._400} marginRight={1} marginLeft={2} />
           <Text textStyle="p medium regular" color={Color.Neutral._700}>
-            {formatTokenAmountWithDecimal(amount, tokenDecimals)} {tokenSymbol}
+            {isFungible(tokenType) && formatTokenAmountWithDecimal(amount, tokenDecimals) + " "}
+            {tokenSymbol}
```

Comment:
is `tokenSymbol` guaranteed to be empty if the tokenType is Fungible? I suggest moving the `tokenSymbol` variable behind the `isFungible` guard clause to ensure there isn't an edge case where the `tokenSymbol` unexpectedly renders.

Reply:
Done.